modules/tinymce/src/plugins/advlist/main/ts/core/ListUtils.ts

```diff
@@ -20,8 +20,16 @@ const getSelectedStyleType = (editor: Editor): Optional<string> => {
   return Optional.from(style);
 };
 
+// Lists/core/Util.ts - Duplicated in Lists plugin
+const isEditableList = (editor: Editor, element: Element): boolean => {
+  const parentList = editor.dom.getParent(element, 'ol,ul,dl');
+  const editableList = editor.dom.getContentEditable(parentList ?? element) === 'true';
+  return editableList || editor.dom.getContentEditableParent(parentList ?? element) !== 'false';
```

Comment:
You've got code duplicate here... Also why are we checking if it is both true and not false here? That doesn't make a lot of sense here... I think this could be simplified down to just this:
```suggestion
  return editor.dom.getContentEditableParent(parentList ?? element) !== 'false';
```

Reply:
Ahh nvm sorry Mitch, I missed one does `getContentEditableParent` and the other just `getContentEditable`. In that case we should just clean up the duplication... So maybe something like this:
```
const isEditableList = (dom: DOMUtils, element: Element): boolean => {
  const parentList =dom.getParent(element, 'ol,ul,dl') ?? element;
  const editableList = dom.getContentEditable(parentList) === 'true';
  return editableList || dom.getContentEditableParent(parentList) !== 'false';
}
```

Reply:
Yep, that's how it was originally. Forgot to revert back when I was trying to fix the Advlist `api.setEnabled` implementation.